src/gatsby-plugin-theme-ui/index.js

```diff
@@ -632,7 +632,7 @@ export default {
     RepoButton: {
```

Comment:
not sure about changes to this file, /CC @rogermparent?

Reply:
It's just a legacy-ish styles file, similar to `tailwind.css.js`. As long as nothing is broken with the consumers of the `RepoButton` and `Switch` styles, these changes are fine.